src/ripple/rpc/handlers/AccountLines.cpp

```diff
@@ -97,13 +97,8 @@ Json::Value doAccountLines (RPC::Context& context)
     std::string strIdent (params[jss::account].asString ());
     AccountID accountID;
 
-    if (auto jv = RPC::accountFromString (accountID, strIdent))
-    {
-        for (auto it = jv.begin (); it != jv.end (); ++it)
-            result[it.memberName ()] = it.key ();
-
+    if ((result = RPC::accountFromString (accountID, strIdent)))
```

Comment:
Yeah.  The extra parentheses were to remove this warning from clang:
```
warning: using the result of an
      assignment as a condition without parentheses [-Wparentheses]
    if (result = RPC::accountFromString (accountID, strIdent))
```
Probably the smart thing to do here is add a comment about the motivation for the extra parentheses.